lib/helpers/contentfulEntry.js

```diff
@@ -136,24 +136,18 @@ async function getTopicTemplates(contentfulEntry) {
     return result;
   }
 
-  templateFieldNames.forEach((fieldName) => {
+  const fields = contentfulEntry.fields;
+  /* eslint-disable */
+  // TODO: Disabling our linter probably isn't best way to handle this...
+  for (const fieldName of templateFieldNames) {
+    const templateTopicRef = fields[`${fieldName}Topic`];
     result[fieldName] = {
```

Comment:
Can we call this `result`,  `topicTemplates`? Made it a bit easier for me once I understood what the result was, even though the function does implicitly refers to the result as such.

Also, a minor ask, the docs at the beginning of the function `getTopicTemplates` should be updated to include the description and param names.